### PR TITLE
Generate a comment on cache manifest

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

File: lib/phoenix/digester.ex [hunk 1]
```diff
@@ -83,15 +83,21 @@ defmodule Phoenix.Digester do
     write_manifest(latest, digests, output_path)
   end
 
+  @comment "This is file was auto-generated by `mix phx.digest`. Remove it and all generated artefacts with `mix phx.digest.clean --all`"
```
Comment (abitdodgy) [inline lib/phoenix/digester.ex:86]: There is a typo there. 

```suggestion
  @comment "This file was auto-generated by `mix phx.digest`. Remove it and all generated artefacts with `mix phx.digest.clean --all`"
```

**Comment by abitdodgy** [comment]
@josevalim you literally merged that a few seconds before I added this suggestion 😀

Care for a pull request?

**Comment by josevalim** [comment]
@abitdodgy yes, please do send a PR :)

**Pull request merged**